master-cloudconfig.yaml

```diff
@@ -0,0 +1,856 @@
+#cloud-config
+hostname: {{.Node.Node.Hostname}}
+ssh_authorized_keys:
+  - "ssh-rsa FILLME user"
+write_files:
+- path: /etc/hosts
+  permissions: 0644
+  owner: root
+  content: |
+    127.0.0.1 localhost
+    127.0.0.1 {{.Node.Node.Hostname}}
```

Comment:
When we work on separating the cloud configs I would like to have a discussion about the structures being injected here. I would like to have it logically structured like our TPRs. When I read `.Node.Node.` it smells already.